packages/ketcher-react/src/script/editor/Editor.ts

```diff
@@ -654,8 +654,13 @@ function domEventSetup(editor: Editor, clientArea: HTMLElement) {
     { target: document, eventName: 'mousemove' },
     { target: document, eventName: 'mouseup' },
     { target: document, eventName: 'mouseleave' },
+    {
+      target: clientArea,
+      eventName: 'mouseleave',
+      toolEventName: 'mouseLeaveClientArea'
+    },
```

Comment:
Maybe we should add a type for these after all? :) Or at least an enum?

Reply:
We do. But I still stand for the need of a separate task for that. Especially now, when it got even more complicated.